core-bundle/src/Framework/ContaoFramework.php

```diff
@@ -80,9 +75,8 @@ class ContaoFramework implements ContaoFrameworkInterface, ContainerAwareInterfa
      */
     private $hookListeners = [];
 
-    public function __construct(RequestStack $requestStack, RouterInterface $router, ScopeMatcher $scopeMatcher, string $rootDir, int $errorLevel)
+    public function __construct(?RequestStack $requestStack /* removed in Contao 4.7 */, RouterInterface $router, ScopeMatcher $scopeMatcher, string $rootDir, int $errorLevel)
```

Comment:
Why did you comment this instead of removing it?

Reply:
Because it would be a BC break?

Reply:
Hm, the parameter has been removed in Contao 4.7 and your pull request is targeted against Contao 4.7. 🤔

Reply:
What do you mean it has been removed?

Reply:
That's what your comment says: `/* removed in Contao 4.7 */`